README.md

```diff
@@ -55,7 +55,7 @@ do you want to permanently mark this value as trusted (y/N)? y
 warning: ignoring untrusted substituter 'https://nix-cache.lowrisc.org/public/', you are not a trusted user.
 ```
 
-For Linux users, there is also an environment with the sonata simulator included:
+There is also an environment with the sonata simulator included:
```

Comment:
Not strictly true just yet but will be true shortly so may as well just change it here.